packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.test.ts

```diff
@@ -139,87 +122,43 @@ describe("MongoDbEmbeddedContentStore", () => {
       "custom-embedded_content"
     );
   });
-  it("has an overridable default collection name", async () => {
+  describe("loadEmbeddedContent", () => {
+    it("loads embedded content belonging to the page provided", async () => {
+      assert(store);
+      const pageEmbeddings = await store.loadEmbeddedContent({ page });
+      const anotherPageEmbeddings = await store?.loadEmbeddedContent({
+        page: anotherPage,
+      });
+      expect(pageEmbeddings[0].sourceName).toBe(page.sourceName);
+      expect(anotherPageEmbeddings[0].sourceName).toBe(anotherPage.sourceName);
+    });
+    it("does NOT load embedded content for some other page", async () => {
+      assert(store);
+      expect(
+        await store.loadEmbeddedContent({
+          page: { ...page, sourceName: "source3" },
+        })
+      ).toStrictEqual([]);
+      expect(
+        await store.loadEmbeddedContent({
+          page: { ...page, url: page.url + "/" },
+        })
+      ).toStrictEqual([]);
+    });
+  });
+  it("updateEmbeddedContent replaces existing embedded content belonging to the page provided with the new embedded content array provided", async () => {
```

Comment:
We have a describe block for `loadEmbeddedContent` and `deleteEmbeddedContent` that has individual test cases within.

Let's use a similar structure here, i.e. a describe block for `updateEmbeddedContent` with this test inside.